core/bentley/src/ProcessDetector.ts

```diff
@@ -37,13 +37,13 @@ export class ProcessDetector {
    * @note this indicates that this is a browser process started by an iTwin mobile application.
    * It will return `false` when running user-launched web browsers on a mobile device.
    */
-  public static get isMobileAppFrontend() { return this.isBrowserProcess && undefined !== (window as any).itwin; }
+  public static get isMobileAppFrontend() { return this.isBrowserProcess && window.location.origin === "imodeljs://app"; }

```

Comment:
will window.location always be defined, on all browsers? Where does "origin" come file (I presume you set it somewhere?)

Reply:
That is part of window.location in all browsers that I know of.